### PR TITLE
feat: Use our official graphql page cursors gem

### PR DESCRIPTION
**Pull request by jonallured**
This PR is the following:

* add `graphql-page_cursors` to the Gemfile
* `$ bundle install`
* `$ replace "Types::Pagination::PageableConnection" "GraphQL::PageCursorConnection" app/graphql`
* `$ be rake`
* ensure that's green
* `$ git rm -rf app/graphql/types/pagination`
* `$ be rake`
* ensure that's green
* `$ git rm -rf spec/requests/api/graphql/queries/submissions_pagination_spec.rb`

So now convection is relieved of the responsibility of testing pagination cursor compliance and instead the gem covers that. Further, adding pagination to other queries is a simple matter of inheriting from `GraphQL::PageCursorConnection`.

### CODE REVIEW

**Review by iskounen — APPROVED** [review]

**Review by pepopowitz — APPROVED** [review]
❤️ ❤️ ❤️

File: Gemfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ gem 'decent_exposure' # for safely referencing variables in views
 gem 'gemini_upload-rails' # for admins to upload images
 gem 'graphiql-rails' # A lovely interface to the API
 gem 'graphql', '1.10.10' # A lovely API
+gem 'graphql-page_cursors'
```
Comment (damassi) [inline Gemfile:27]: Where did this come from?

Comment (jonallured) [reply Gemfile:27]: It's our hackathon project! 🎉 

https://github.com/artsy/graphql-page_cursors

Comment (damassi) [reply Gemfile:27]: Oh nice, thats what i was thinking but was also wondering if this had already been done 👍

**Pull request merged**